Source/Classes/PINRemoteImageManager.m

```diff
@@ -233,6 +245,7 @@ - (nonnull instancetype)initWithSessionConfiguration:(nullable NSURLSessionConfi
 }
 
 - (id<PINRemoteImageCaching>)defaultImageCache {
+    NSAssert(NO, @"Assert should not have been reached");
```

Comment:
Actually, I don't think this can be asserted because I think it would be legitimate for a subclass to override the (deprecated) instance method and call `[super default...]` with the intent of customizing it.

Reply:
Good point. Updated!